fmcapi/fmc.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         logging_level="INFO",
         debug=False,
         limit=1000,
-        timeout=5,
+        timeout=30,
```

Comment:
Why this change? Unrelated to the PR and you as the user can just pass in the Timeout=30 when you init the object?